packages/web/src/common/store/tipping/sagas.ts

```diff
@@ -252,28 +238,23 @@ function* confirmTipIndexed({
     )
     try {
       const sdk = yield* getSDK()
+      const { data } = yield* call([sdk.full.tips, sdk.full.tips.getTips], {
+        txSignatures: [signature]
+      })
 
-      const senderUserId = Id.parse(sender.user_id)
-
-      const { data } = yield* call(
-        [sdk.full.users, sdk.full.users.getSupporter],
-        {
-          id: Id.parse(recipient.user_id),
-          supporterUserId: senderUserId,
-          userId: senderUserId
-        }
-      )
-
-      if (data) {
-        console.debug('Tip indexed')
+      if (data && data.length > 0) {
+        console.info('Tip indexed')
         return true
+      } else if (data?.length === 0) {
+        console.debug('Tip not indexed yet...')
       }
+      yield* delay(delayMs)
     } catch (e) {
       console.error('Error confirming tip indexed: ', e)
+      return false
     }
-    yield* delay(delayMs)
   }
-  console.error('Tip could not be confirmed as indexed')
+  console.error('Tip could not be confirmed as indexed before timing out.')
```

Comment:
nit: this reads a little weird. how about just "Tip could not be confirmed before timeout."

bc is it "tip could not be confirmed as indexed, before timing out" or "tip could not be confirmed, as indexed before timing out"? i know which but without context it could be confusing.